integrations/lib/plugindata/access_request.go

```diff
@@ -111,5 +115,7 @@ func EncodeAccessRequestData(data AccessRequestData) (map[string]string, error)
 		}
 		result["system_annotations"] = string(annotaions)
 	}
+
+	result["suggested_reviewers"] = strings.Join(data.SuggestedReviewers, ",")
```

Comment:
Is it ok to encode the reviewers like this?

Encoding things like this can cause subtle bugs
Can we do the Json marshal instead? Or are we safe here?